examples/graph/wgan-graphs.py

```diff
@@ -0,0 +1,601 @@
+"""
+
+
+def GraphGenerator(
+    dense_units, dropout_rate, latent_dim, adjacency_shape, feature_shape,
+):
+    z = keras.layers.Input(shape=(LATENT_DIM,))
+    # Propagate through one or more densely connected layers
+    x = z
+    for units in dense_units:
+        x = keras.layers.Dense(units, activation="tanh")(x)
+        x = keras.layers.Dropout(dropout_rate)(x)
+
+    # Map outputs of previous layer (x) to [continuous] adjacency tensors (x_adjacency)
+    x_adjacency = keras.layers.Dense(tf.math.reduce_prod(adjacency_shape))(x)
+    x_adjacency = keras.layers.Reshape(adjacency_shape)(x_adjacency)
+    # Symmetrify tensors in the last two dimensions
+    x_adjacency = (x_adjacency + tf.transpose(x_adjacency, (0, 1, 3, 2))) / 2
+    x_adjacency = keras.layers.Softmax(axis=1)(x_adjacency)
+
+    # Map outputs of previous layer (x) to [continuous] feature tensors (x_features)
+    x_features = keras.layers.Dense(tf.math.reduce_prod(feature_shape))(x)
+    x_features = keras.layers.Reshape(feature_shape)(x_features)
+    x_features = keras.layers.Softmax(axis=2)(x_features)
+
+    return keras.Model(inputs=z, outputs=[x_adjacency, x_features], name="Generator")
+
+
+generator = GraphGenerator(
+    dense_units=[128, 256, 512],
+    dropout_rate=0.2,
+    latent_dim=LATENT_DIM,
+    adjacency_shape=(BOND_DIM, NUM_ATOMS, NUM_ATOMS),
+    feature_shape=(NUM_ATOMS, ATOM_DIM),
+)
+generator.summary()
+
+"""
+## Graph discriminator
+
+
+**Graph convolutional layer**. The [relational graph convolutional
+layers](https://arxiv.org/abs/1703.06103) implements non-linearly transformed
+neighborhood aggregations. We can define these layers as follows:
+
+`H^{l+1} = σ(D^{-1} @ A @ H^{l+1} @ W^{l})`
+
+
+Where `σ` denotes the non-linear transformation (commonly a ReLU activation), `A` the
+adjacency tensor, `H^{l}` the feature tensor at the `l:th` layer, `D^{-1}` the inverse
+diagonal degree tensor of `A`, and `W^{l}` the trainable weight tensor at the `l:th`
+layer. Specifically, for each bond type (relation), the degree tensor expresses, in the
+diagonal, the number of bonds attached to each atom. Notice, in this tutorial `D^{-1}` is
+omitted, for two reasons: (1) it's not obvious how to apply this normalization on the
+continuous adjacency tensors (generated by the generator), and (2) the performance of the
+WGAN without normalization seems to work just fine. Furthermore, in contrast to the
+[original paper](https://arxiv.org/abs/1703.06103), no self-loop is defined, as we don't
+want to train the generator to predict "self-bonding".
+
+
+
+"""
+
+
+class RelationalGraphConvLayer(keras.layers.Layer):
+    def __init__(
+        self,
+        units=128,
+        activation="relu",
+        use_bias=False,
+        kernel_initializer="glorot_uniform",
+        bias_initializer="zeros",
+        kernel_regularizer=None,
+        bias_regularizer=None,
+        **kwargs
+    ):
+        super(RelationalGraphConvLayer, self).__init__(**kwargs)
```

Comment:
Every time you call `super` in this example, use the Python convention, `super().__init__(**kwargs)`

Reply:
Notice, I modified the super calls (as per your suggestion) for the other classes too. Not sure if that's what you asked for.